google-cloud-storage/src/test/java/org/jclouds/googlecloudstorage/blobstore/integration/GoogleCloudStorageBlobIntegrationLiveTest.java

```diff
@@ -94,15 +94,38 @@ public void testCreateBlobWithExpiry() throws SkipException {
    }
 
    @Override
-   @Test(enabled = false)
+   @Test(groups = { "integration", "live" })
    public void testFileGetParallel() throws SkipException {
-      // Implement Parallel uploads
+      throw new SkipException("Implement Parallel uploads");
+      // TODO: Implement Parallel uploads
    }
 
    @Override
-   @Test(enabled = false)
+   @Test(groups = { "integration", "live" })
    public void testPutFileParallel() throws SkipException {
-      // Implement Parallel uploads
+      throw new SkipException("Implement Parallel uploads");
+      // TODO: Implement Parallel uploads
```

Comment:
Why do we disable this test?  It passes when running it locally.  I see these failures:

```
Failed tests:
  GoogleCloudStorageBlobIntegrationLiveTest>BaseBlobIntegrationTest.testGetRangeOutOfRange:376 Invalid range not caught
  GoogleCloudStorageBlobIntegrationLiveTest>BaseBlobIntegrationTest.testMultipartUploadMultipleParts:919 » UnsupportedOperation
  GoogleCloudStorageBlobIntegrationLiveTest>BaseBlobIntegrationTest.testMultipartUploadNoPartsAbort:859 » UnsupportedOperation
  GoogleCloudStorageBlobIntegrationLiveTest>BaseBlobIntegrationTest.testMultipartUploadSinglePart:885 » UnsupportedOperation
  GoogleCloudStorageBlobIntegrationLiveTest>BaseBlobIntegrationTest.testPutInputStream:599->BaseBlobIntegrationTest.testPut:655 » HttpResponse
  DefaultObjectAccessControlsApiLiveTest.testCreateDefaultObjectAcl:49->createBucket:43 » HttpResponse
  ResumableUploadApiLiveTest.testResumableChunkedUpload:140 » HttpResponse comma...
  ObjectApiLiveTest.deleteBucket:454 » IllegalState {
```